improver_tests/acceptance/test_weighted_blending.py

```diff
@@ -155,7 +155,6 @@ def test_invalid_nonlin_lin(tmp_path):
         run_cli(args)
 
 
-@pytest.mark.xfail(run=False, reason="takes ~5 minutes to run")
 def test_percentile(tmp_path):
```

Comment:
There are a few instances of `@pytest.mark.slow` in this file that can now be removed, as all tests take less than 1 second.

Reply:
Thanks Gavin, good spot.